doc/docs/master/reference/pipeline_spec.md

```diff
@@ -421,6 +425,18 @@ nothing to process. For more information about scheduling GPUs see the
 [Kubernetes docs](https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/)
 on the subject.
 
+### Sidecar Resource Limits (optional)
+
+`sidecar_resource_limits` determines the upper threshold of resources
+allocated to the sidecar, init, or storage containers. You can also
```

Comment:
`sidecar_resource_limits` is only for the sidecar - the init and storage containers use `resource_limits`.

Reply:
corrected